### PR TITLE
Move BS to Rococo and Zeitgeist to Polkadot

### PR DESCRIPTION
**Pull request by sea212**
* Adjust chainspecs and clients

* Bump runtime version and remove migrations

* Use DOT in Zeitgeist XCM integration tests

* Use Rococo in Battery Station XCM tests

### CODE REVIEW

**Comment by sea212** [comment]
This PR just merges the new Polkadot configuration from the branch `release-v0.3.8-polkadot`, that already was improved, into main. The only new changes are the merge of main into the branch and formatting, see https://github.com/zeitgeistpm/zeitgeist/compare/9b3a865bba6efe36bc2e3dee64e9f114ef140a70...release-v0.3.8-polkadot

File: runtime/common/src/lib.rs [hunk 1]
```diff
@@ -52,17 +52,6 @@ macro_rules! decl_common_types {
...
-        pub type Executive = frame_executive::Executive<
-            Runtime,
-            Block,
-            frame_system::ChainContext<Runtime>,
-            Runtime,
-            AllPalletsWithSystem,
-            zrml_prediction_markets::migrations::AddOutsiderBond<Runtime>,
-        >;
-
-        #[cfg(not(feature = "parachain"))]
```
Comment (Chralt98) [inline runtime/common/src/lib.rs:65]: Don't we want the migrations from now on?

**Review by Chralt98 — COMMENTED** [review]
Looks okay.

File: runtime/common/src/lib.rs (hunk 1 above)
Comment (sea212) [reply runtime/common/src/lib.rs:65]: [The migration is still included](https://github.com/zeitgeistpm/zeitgeist/pull/992/files/021803683fd5b1438e7b89a15406e7bc84383499#diff-3287a4ee060b1070a8e6559a094b4890a596a1b281effef54cafe59da20edf49L72), the distinction between the standalone and parachain environment was just removed to deduplicate the code, as it does not matter for the current migration.

File: runtime/battery-station/Cargo.toml [hunk 2]
```diff
@@ -148,7 +148,7 @@ parachain = [
 
     # XCM
 
-    "kusama-runtime",
+    "rococo-runtime",
```
Comment (maltekliemann) [inline runtime/battery-station/Cargo.toml:151]: These seem to be ordered lexicographically, but `"rococo-runtime"` is not sorted in correctly.

File: runtime/battery-station/Cargo.toml [hunk 3]
```diff
@@ -173,7 +173,7 @@ runtime-benchmarks = [
     "frame-system-benchmarking",
     "frame-system/runtime-benchmarks",
     "hex-literal",
-    "kusama-runtime?/runtime-benchmarks",
```
Comment (maltekliemann) [inline runtime/battery-station/Cargo.toml:176]: These seem to be ordered lexicographically, but `"rococo-runtime"` is not sorted in correctly.

**Review by Chralt98 — DISMISSED** [review]

File: runtime/battery-station/Cargo.toml (hunk 2 above)
Comment (sea212) [reply runtime/battery-station/Cargo.toml]: 67906fc019237c8c122e016c8d4f02d0052b4926

File: runtime/battery-station/Cargo.toml (hunk 3 above)
Comment (sea212) [reply runtime/battery-station/Cargo.toml:176]: 67906fc019237c8c122e016c8d4f02d0052b4926

Comment (sea212) [reply runtime/battery-station/Cargo.toml:176]: 67906fc019237c8c122e016c8d4f02d0052b4926

**Review by maltekliemann — APPROVED** [review]

**Pull request merged**